### PR TITLE
UIApplication Reactive Extensions

### PR DESCRIPTION
**Pull request by cocoahero**
#### Checklist
- [x] Updated CHANGELOG.md.

### CODE REVIEW

**Comment by cocoahero** [comment]
If anyone is able to help write a spec for this, it would be appreciated. The current spec that is written crashes, most likely because `UIApplication.shared` isn't available.

**Comment by andersio** [comment]
@mdiep Mocking `UIApplication` seems impossible through normal means. Should we drop the unit test, or introduce some sort of `UIApplicationProtocol` to replace the direct extension constraint to `UIApplication`?

**Comment by mdiep** [comment]
I think you can use `UIApplication.shared` if you add a test host app.

But I'd be okay dropping the unit test in this case.

**Comment by cocoahero** [comment]
@andersio @mdiep I have removed the unit test case. In terms of resolving the project conflict, would you prefer a rebase or a merge?

**Comment by mdiep** [comment]
Either is fine 🙂

**Comment by cocoahero** [comment]
This should be ready to go now. 🙂

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,9 @@
 2. Added `title` binding target to `UIViewController` (#3588, kudos to @cocoahero).
 3. Added several trigger signals for view lifecycle events to `UIViewController` (#3588, kudos to @cocoahero).
 
+# 7.X.0
+1. Added `applicationIconBadgeNumber` binding target to `UIApplication` (#3589, kudos to @cocoahero).
```
Comment (mdiep) [inline CHANGELOG.md]: This needs to go at the top under `master`. 🙈

Comment (cocoahero) [reply CHANGELOG.md]: 😅 Fixed!

**Comment by mdiep** [comment]
Awesome, thanks!

**Pull request merged**